gdscript-customization.el

```diff
@@ -129,6 +129,16 @@ so it is not slowing down Godot execution."
   :type 'boolean
   :group 'gdscript)
 
+(defcustom gdscript-docs-force-online-lookup nil
+  "the string to determine if we are browsing online API to allow filtering in EWW"

```

Comment:
The docstring should be something like

> If `true`, calling commands like `gdscript-docs-browse-api` browses the online API reference, even if a local copy is available.

Reply:
I'm not quite sure what happened to my brain when I typed that ;) I'll change it